package.json

```diff
@@ -401,8 +401,7 @@
             {
                 "key": "ctrl+shift+f",
                 "mac": "cmd+shift+f",
-                "command": "workbench.view.search",
-                "when": "!searchViewletVisible",
+                "command": "workbench.action.findInFiles",
```

Comment:
ctrl+shift+f would be the VSCode default for findInFiles anyways, so it would not really be necessary that it's set in this keybinding again, but i figured it's a bit more explicit and resilient against future changes

Reply:
agree